src/QsFmt/App/App.fsproj

```diff
@@ -6,7 +6,7 @@
     <RootNamespace>Microsoft.Quantum.QsFmt.App</RootNamespace>
     <!--
     The MsBuild imports cause a bunch of MSB3277 errors that seem benign.
-    I believe the root is the use of the CommandLineParser that seems to target netstandard2.0
+    I believe the root is the use of the CommandLineParser that seems to target net6.0
```

Comment:
Same as my other comment - it would seem that this update is unnecessary

Reply:
Agreed, thanks. I reverted the change.